db/schema.rb

```diff
@@ -51,5 +51,13 @@
   add_index "facts_bookings", ["dimensions_date_id"], name: "index_facts_bookings_on_dimensions_date_id", using: :btree
   add_index "facts_bookings", ["reference_number"], name: "index_facts_bookings_on_reference_number", unique: true, using: :btree
 
+  create_table "imports", force: :cascade do |t|
+    t.string   "importer"
+    t.integer  "inserted_records"
+    t.jsonb    "log"
```

Comment:
This is the first steps towards created an audit dimension:

When a fact table row is created in the ETL back room, it is helpful to create an audit dimension
containing the ETL processing metadata known at the time. A simple audit dimension row could
contain one or more basic indicators of data quality, perhaps derived from examining an error event schema that records data quality violations encountered while processing the data. Other useful audit dimension attributes could include environment variables describing the versions of ETL code used to create the fact rows or the ETL process execution time stamps. These environment variables are especially useful for compliance and auditing purposes because they enable BI tools to drill down to determine which rows were created with what versions of the ETL software.

I will rethink how this is being done and update the code.